spec/enumerables_spec.rb

```diff
@@ -0,0 +1,129 @@
+require_relative '../enumerables.rb'
+
+describe Enumerable do
+  let(:array) { [1, 2, 3, 4, 5, 6, 7, 8, 9] }
+  let(:words_array) { %w[ant bear cat] }
+  let(:nil_true) { [nil, true, 99, false] }
+  let(:proced) { proc { |n| n * 2 } }
+  describe '#my_each' do
+    it 'Returns each element' do
+      expect(array.my_each { |n| n * n }).to eql(array)
```

Comment:
It actually returns the array itself. I think you can make the test more solid by comparing it with the actual each. I did it in the below format. let me know if you don't like it.
```ruby
it 'Returns the array itself as the actual each' do
      expect(array.my_each { |n| n * n }).to eq(array.each { |n| n * n })
end
```

Reply:
Done! Thank you for pointing it out! We changed it as suggested